modules/python/src2/hdr_parser.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+##!/usr/bin/env python
```

Comment:
Sure, will do. Please also let know where I can find tensorflow_inception_graph.pb on the autobuild servers? :) I'm using OPENCV_DNN_TEST_DATA_PATH variable and it gives me /opt/build-testdata. You can check the junit report [here](http://pullrequest.opencv.org/buildbot/builders/precommit_macosx/builds/11255/steps/test_java/logs/junit-report-html#DnnTensorFlowTest). But it is not there ))